chronosphere/notification_policy_data.go

```diff
@@ -178,9 +185,21 @@ func notifierListMapToModel(notifierList RoutesNotifierList) (*configmodels.Rout
 	return &configmodels.RoutesNotifierList{
 		NotifierSlugs:      sliceutil.Map(notifierList.Notifiers, func(n *RoutesNotifierListNotifier) string { return n.Slug }),
 		RepeatIntervalSecs: durationInSecs,
+		GroupBy:            notifierListToGroupByModel(notifierList),
 	}, nil
 }
 
+func notifierListToGroupByModel(notifierList RoutesNotifierList) *configmodels.NotificationPolicyRoutesGroupBy {
+	if notifierList.GroupBy == nil {
+		return nil
+	}
+	routesGroupBy := &configmodels.NotificationPolicyRoutesGroupBy{}
+	if notifierList.GroupBy.LabelNames != nil {
```

Comment:
since this is a slice, unless you explicitly want to differentiate `nil` and an empty slice, generally it's better to do `if len(labelNames) > 0` rather than comparing to nil.

however in this case, we can simplify this even further (see other suggestion)